src/main/java/core/basesyntax/model/FruitTransactionDto.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+import com.opencsv.bean.CsvBindByPosition;
```

Comment:
You don't have to use an additional library. Youll be enough with java io and nio...

Reply:
I used the OpenCSV library as it significantly reduces the number of steps and lines of code. If these recommendations need to be followed, I will change my code. But I would like to hear opinion of the mentor too.